tests/Integration/Maker/MakeFactoryTest.php

```diff
@@ -118,6 +119,22 @@ public function can_create_factory_in_test_dir(): void
         $this->assertFileExists(self::tempFile('tests/Factory/CategoryFactory.php'));
     }
 
+    /**
+     * @test
+     */
+    public function can_create_factory_in_test_dir_with_nested_factory_already_created(): void
+    {
+        if (!\getenv('DATABASE_URL')) {
+            self::markTestSkipped('doctrine/orm not enabled.');
+        }
+
+        $tester = $this->makeFactoryCommandTester();
+        $tester->execute(['class' => Contact::class, '--test' => true]);
+        $tester->execute(['class' => Address::class, '--test' => true]);
+        $this->assertFileExists(self::tempFile('tests/Factory/ContactFactory.php'));
+        $this->assertFileExists(self::tempFile('tests/Factory/AddressFactory.php'));
```

Comment:
I think you can change your code to this:
```suggestion
        $tester = $this->makeFactoryCommandTester();
        
        $tester->execute(['class' => Contact::class, '--test' => true]);    
        
        $this->assertFileExists(self::tempFile('tests/Factory/ContactFactory.php'));
        $this->assertFileExists(self::tempFile('tests/Factory/AddressFactory.php'));
        
        // here, we're faking multiple calls to `bin/console make:factory`
        // so it's perfectly acceptable to manually load the newly created class
        require_once self::tempFile('tests/Factory/AddressFactory.php';
        
        $tester->execute(['class' => Address::class, '--test' => true]);
```